app/assets/stylesheets/content/ui_select.sass

```diff
@@ -1,6 +1,7 @@
 // keep this rule if you want ui-select drag-n-drop to work properly
-.ui-select-match .ui-select-match-item *
-  pointer-events: none
+.ui-select-match .ui-select-match-item
+  span, span *
+    pointer-events: none
```

Comment:
Can someone have a second look at that rule? (CC @myabc @floriank )

To me the `span *` part looks suspicious... Doesn't that match for **all** elements that are somewhere inside a `span`?